examples/siro_sandbox/controllers/fetch_baselines_controller.py

```diff
@@ -81,6 +81,7 @@ def __init__(
         gym_env,
         habitat_env,
         hybrid_social_nav,
+        oracle_follow,
```

Comment:
I always like boolean parameter names to sound boolean. How about `force_oracle_follow` or `use_oracle_follow`.

Reply:
Done